.cirrus.yml

```diff
@@ -2,9 +2,15 @@ environment:
   DOCKER_USER_NAME: ENCRYPTED[5401d6fcedc8f4252e876506a88d910ff6cf161c3fbabd75cafca088e2c05cd7c64bf30ac54e37fa29a26d085349fdf9]
   DOCKER_PASSWORD: ENCRYPTED[955ccf4734b017d3880cf99d346b07435245666e095054b87026b99bad749e17134999986cfd109ace0c71e5a7025da6]
 
-env:
-  FLUTTER_VERSION: 1.2.1
-
 flutter_docker_builder:
+  env:
+    matrix:
+    - FLUTTER_VERSION: v1.2.1 # stable
+    - FLUTTER_VERSION: stable
+    - FLUTTER_VERSION: v1.5.4 # beta
+    - FLUTTER_VERSION: beta
+    - FLUTTER_VERSION: v1.5.8 # dev
+    - FLUTTER_VERSION: dev
+    - FLUTTER_VERSION: master
```

Comment:
Seems you always end up with two tags for a single image (`stable` is `v1.2.1`, `beta` is `1.5.4`, etc.).

What do you think about having two environment variables: `FLUTTER_BRANCH` and `FLUTTER_TAG`. Then you can have matrix like this:

```
matrix:
    - FLUTTER_BRANCH: v1.2.1 # stable
       FLUTTER_TAG: stable
    - FLUTTER_BRANCH: v1.5.4 # beta
       FLUTTER_TAG: beta
    - FLUTTER_BRANCH: v1.5.8 # dev
       FLUTTER_TAG: dev
    - FLUTTER_BRANCH: master
       FLUTTER_TAG: latest
```

Reply:
agree, we can reduce the amount of builds by almost half, and avoid weird condition check